### ISSUE TITLE
Fix PhET menu focus highlight

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/joist#444]
Wile working on phetsims/scenery#676, @zepumph and I noticed that the bottom of the PhET menu button focus highlight is clipped off screen for default layouts.  We should fix this, probably with a custom focus highlight using ```FocusHighlightPath```.

**Comment by zepumph** [comment]
Here is an example from FL
![image](https://user-images.githubusercontent.com/6856943/31205937-1573fe56-a921-11e7-867f-f6e72121af20.png)

**Comment by zepumph** [comment]
@terracoda could you confirm that this is a problem to have the bottom cut off from a design standpoint?

**Comment by terracoda** [comment]
@zepumph, oh that's an issue. We had that issue in JT just before release, and @jessegreenberg fixed it. I do not know if the solution was a custom one.

**Comment by zepumph** [comment]
Thanks! We'll take care of it.

**Comment by jessegreenberg** [comment]
This is also an issue for the KeyboardHelpButton.

![capture](https://user-images.githubusercontent.com/6396244/31239414-3b88f7c4-a9ba-11e7-90f2-1e9ea4571f97.PNG)

We used to have a custom focus highlight around these, we were hoping they could be removed after phetsims/scenery#676, but they might need a bit of adjustment.

**Comment by jessegreenberg** [comment]
Marking high priority, this should be done before  an RC with keyboard nav.

**Comment by jessegreenberg** [comment]
![capture](https://user-images.githubusercontent.com/6396244/31363281-9dd10dd2-ad1a-11e7-9bee-72ad2b281991.PNG)

Fixed in the above commit, thanks for the help @zepumph!
